show_client/interface_transceiver_cli.go

```diff
@@ -0,0 +1,31 @@
+package show_client
+
+import (
+	log "github.com/golang/glog"
+	sdc "github.com/sonic-net/sonic-gnmi/sonic_data_client"
+)
+
+func getTransceiverErrorStatus(options sdc.OptionMap) ([]byte, error) {
+	var intfs []string
+	if interfaces, ok := options["interfaces"].Strings(); ok {
```

Comment:
I think perhaps we can register showCmdOptionInterface instead of showCmdOptionInterfaces and then use options["interface"].String, so we can simplify the check since we will be dealing with single string and not array.